group_vars/dataspace/dev.yml

```diff
@@ -0,0 +1,54 @@
+---
+dspace_install_path: "/mnt/disks/{{ project }}{{ env }}datastore/prod-copy"
+dspace_install_directory: "{{ dspace_install_path }}/dspace"
+env: "dev"
+project: "dataspace"
+nginx_type: opensource
+nginx_install_from: nginx_repository
+deploy_user: "dspace"
+deploy_user_uid: 1005
+java_type: jdk
+tomcat_user: "{{ deploy_user }}"
+tomcat_group: "{{ deploy_user }}"
+tomcat_install_path: "/home/{{ deploy_user }}"
+dspace_db_user: "oai_db_user"
+dspace_db_password: "{{ vault_dataspace_dev_db_password }}"
+dspace_db_name: "oai_staging_db"
```

Comment:
I approved of this for the OAR with https://github.com/pulibrary/princeton_ansible/pull/1624, but I just to confirm, there isn't going to conflict with the planned `staging` environment, correct?

Reply:
it's been a mistake since we implemented this. and a confusing one...